blocks-config/table-of-content/class-uagb-table-of-content.php

```diff
@@ -425,6 +433,75 @@ public function toc_recursive_reusable_heading( $blocks_array ) {
 			return $final_reusable_array;
 		}
 
+		/**
+		 * Sets the current post for usage in template blocks.
+		 *
+		 * @since x.x.x
+		 * @access public
+		 *
+		 * @return WP_Post|null The post if any, or null otherwise.
+		 */
+		public function get_post_from_context() {
+			// Without this temporary fix, an infinite loop can occur where
+			// posts with post content blocks render themselves recursively.
+			if ( is_admin() || defined( 'REST_REQUEST' ) ) {
+				return null;
+			}
+			// Prevent an issue with the terms and conditions page in the woocommerce checkout.
+			if ( class_exists( 'woocommerce' ) && is_checkout() ) {
+				return '';
```

Comment:
This function states that `@return WP_Post|null`.

Hence this line should probably be:
 ```suggestion
				return null;
```

Reply:
Done